### PR TITLE
Set operator_coded from phone number

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

**Comment by lukechurch** [comment]
Also requires @ParthaMoman to review as we're adding a code scheme

File: project_redss/auto_code_surveys.py [hunk 1]
```diff
@@ -48,17 +48,21 @@ def auto_code_surveys(user, data, phone_uuid_table, coda_output_dir):
...
-        #
-        #     td.append_data(
-        #         {"operator": operator},
-        #         Metadata(user, Metadata.get_call_location(), time.time())
-        #     )
-        #
+        # Set operator from phone number
+        for td in data:
+            operator_clean = PhoneCleaner.clean_operator(phone_uuid_table.get_phone(td["uid"]))
+            if operator_clean == Codes.NOT_CODED:
```
Comment (lukechurch) [inline project_redss/auto_code_surveys.py:54]: Please can you monitor as to how often this pattern of needing to switch between CodeID and Control code occurs, and I might revisit the decision to not put matching for Control codes. A few instances isn't sufficient to justify but if it becomes very common boiler plate we might look again.

**Review by lukechurch — APPROVED** [review]

File: project_redss/auto_code_surveys.py (hunk 1 above)
Comment (as2388) [reply project_redss/auto_code_surveys.py:54]: I think this is the only time I've seen it. It's certainly not come up a lot

**Comment by as2388** [comment]
Code scheme was reviewed in #11 so need for another review from @ParthaMoman. Merging

**Comment by as2388** [comment]
Sorry, that should have read "*no need* for another review" from @ParthaMoman

**Pull request merged**